### PR TITLE
feat: create orderProduct model

### PR DESCRIPTION
**Pull request by jkayen**

### CODE REVIEW

File: db/models/orderProduct.js [hunk 1]
```diff
@@ -2,8 +2,22 @@
 
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
 const bcrypt = require('bcryptjs')
-    , {STRING, VIRTUAL, BOOLEAN, INTEGER, DATE} = require('sequelize')
+    , {STRING, VIRTUAL, BOOLEAN, INTEGER, DATE, FLOAT} = require('sequelize')
```
Comment (ekotwick) [inline db/models/orderProduct.js]: unnecessary codes: STRING, VIRTUAL, BOOLEAN, DATE

File: db/models/orderProduct.js [hunk 2]
```diff
@@ -2,8 +2,22 @@
...
-
+  quantity: {
+    type: INTEGER
+    // can we set default from the front end?
+    // defaultValue: 1
+  },
+  unitPrice: {
+    type: FLOAT(2)
+    // we can add this from the front end. Otherwse:
+    // set: this.setDataValue('unitPrice', this.product.price)
```
Comment (ekotwick) [inline db/models/orderProduct.js:16]: not sure that 'this.product.price' should work. _this_ is the orderProduct object.

File: db/models/orderProduct.js [hunk 3]
```diff
@@ -2,8 +2,22 @@
...
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
 const bcrypt = require('bcryptjs')
-    , {STRING, VIRTUAL, BOOLEAN, INTEGER, DATE} = require('sequelize')
+    , {STRING, VIRTUAL, BOOLEAN, INTEGER, DATE, FLOAT} = require('sequelize')
 
 module.exports = db => db.define('orderProduct', {
-
+  quantity: {
+    type: INTEGER
+    // can we set default from the front end?
```
Comment (eyezick) [inline db/models/orderProduct.js:10]: front-end can set a default value in the form, but it won't go down to the database of course unless its submitted

**Review by ekotwick — CHANGES_REQUESTED** [review]
see my comments

**Pull request merged**